jina/types/arrays/memmap.py

```diff
@@ -279,9 +282,13 @@ def _get_doc_array_by_slice(self, s: slice):
     @property
     def _mmap(self) -> 'mmap':
         if self._last_mmap is None:
-            self._last_mmap = mmap.mmap(
-                self._body_fileno, length=0, prot=mmap.PROT_READ
+            self._last_mmap = (
+                mmap.mmap(self._body_fileno, length=0)
+                if __windows__
+                else mmap.mmap(self._body_fileno, length=0, prot=mmap.PROT_READ)
             )
+        if __windows__:
+            self._body.seek(self._start)
```

Comment:
Should this be here?

Reply:
Yes, actually mmap seeks _body to 0 on windows